continue-reading.html

```diff
@@ -6,8 +6,104 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+    <!DOCTYPE html>
+    <html>
+
+    <head>
+      <title>Blog Post</title>
+      <meta charset="utf-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    </head>
+    <style>
+      h1 {
+        text-align: center;
+        font: size 60px;
+        font-family: "Notable", sans-serif;
+      }
+
+      .home-btn {
+        background-color: purple;
+        color: white;
+        cursor: pointer;
+      }
+
+      .home-btn:active {
+        color: blue
+      }
+
+      .home-btn:hover {
+        background-color: rgb(63, 61, 61);
+      }
+    </style>

```

Comment:
These styles could be put into your CSS file to make things a bit more organized